tests/ui/single_char_push_str.fixed

```diff
@@ -0,0 +1,10 @@
+// run-rustfix
+#![warn(clippy::single_char_push_str)]
+
+fn main() {
+    let mut string = String::new();
```

Comment:
We could add the following tests:
- An empty string
- A string with more than 1 character
- A raw string
- A string with a 7 bit unicode point
- A string with a 24 bit unicode point